guides/micronaut-mqtt/micronaut-mqtt.adoc

```diff
@@ -4,23 +4,7 @@ common:requirements.adoc[]
 
 common:completesolution.adoc[]
 
-common:mosquitto.adoc[]
-
-For example, in MacOS:
-
-[source,bash]
-----
-/usr/local/opt/mosquitto/sbin/mosquitto -c /usr/local/etc/mosquitto/mosquitto.conf
-
-1646389659: mosquitto version 2.0.14 starting
-1646389659: Config loaded from /usr/local/etc/mosquitto/mosquitto.conf.
-1646389659: Starting in local only mode. Connections will only be possible from clients running on this machine.
-1646389659: Create a configuration file which defines a listener to allow remote access.
-1646389659: For more details see https://mosquitto.org/documentation/authentication-methods/
-1646389659: Opening ipv4 listen socket on port 1883.
-1646389659: Opening ipv6 listen socket on port 1883.
-1646389659: mosquitto version 2.0.14 running
-----
+common:test-resources-hivemq.adoc[]
```

Comment:
@timyates  should this be mosquitto again?

Reply:
Yeah, done in https://github.com/micronaut-projects/micronaut-guides/pull/1064/commits/7ef54ad2d651073cc4bbdda998ea05893530cbbe